cache-config/t3c-generate/plugin/plugin.go

```diff
@@ -59,7 +58,7 @@ type Plugins interface {
 func AddPlugin(priority uint64, funcs Funcs) {
 	_, filename, _, ok := runtime.Caller(1)
 	if !ok {
-		fmt.Println(time.Now().Format(time.RFC3339Nano) + " Error plugin.AddPlugin: runtime.Caller failed, can't get plugin names") // print, because this is called in init, loggers don't exist yet
+		log.Errorln(time.Now().Format(time.RFC3339Nano) + " Error plugin.AddPlugin: runtime.Caller failed, can't get plugin names")
```

Comment:
This needs to still be a Print, because like the comment said, the loggers haven't been initialized yet

Reply:
Changed back to print and also changed other log statement to print. Moved comment to top of func for readability.